### DISCUSSION TITLE
GitHub and GitLab support no longer depreciated?

### DISCUSSION THREAD

**Discussion by space-bob** [GetPublii/Publii#1648] [Help] [answered]
In versions 0.45.x of Publii the GitHub and GitLab connectors were marked as being depreciated meaning they would probably be removed in the next major release -

![Picture1](https://github.com/user-attachments/assets/f489f56e-e99a-483d-aa7a-5290d76165f9)

In versions 0.46.x it looks like they are no longer depreciated.  Did something change that I missed?  Is it safe to use these?

Thanks

**Comment by dziudek** [comment, marked as answer]
We have observed that use of Git repository is problematic for many users, that's why we have decided to continue support integration via API and we will develop these sync methods in the future which are still convenient for small websites

**Reply by space-bob** [reply]
Thank you.

**Reply by matja31** [reply]
Hello Tomasz, I'm a bit confused by the wording. Do I understand correctly that the option to sync via GitHub will stay in the future versions?

**Reply by dziudek** [reply]
@matja31 - in short - yes :)

**Comment by IO101010** [comment]
Hi @dziudek. When you say that you've noted issues with git repo use, are you referring to issues like this? https://github.com/GetPublii/Publii/discussions/1689

Edit: I see you meant something a little different. Any feedback on the other issue would be appreciated. If GitLab support has been abandoned it would be helpful to know.
